docker/opbeans/node/Dockerfile

```diff
@@ -2,10 +2,6 @@ FROM opbeans/opbeans-node:latest
 
 ENV NODE_ENV=production
 
-RUN npm install pm2 -g
-
 COPY entrypoint.sh /app/entrypoint.sh
-COPY processes.config.js /app/processes.config.js
 
-CMD ["pm2-runtime", "processes.config.js"]
```

Comment:
As per [comment in opbeans-node](https://github.com/elastic/opbeans-node/pull/36#discussion_r219741691), I suggest we leave this line in but change it to use a `ecosystem-workload.config.js` config file.

Reply:
fixed.